deploy/chart/templates/0000_50_olm_02-services.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if .Values.monitoring.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: olm-operator-metrics
+  namespace: {{ .Values.namespace }}
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: olm-operator-serving-cert
```

Comment:
Do we want to remove any OCP-specific annotations? IIRC, this annotation has graduated to beta as well.

Reply:
I think so, yes. There's some other work that has to happen to make enabling metrics work out-of-the-box on upstream though.